### PR TITLE
2852/new flagged management page

### PR DESCRIPTION
**Pull request by YazeedLoonat**
## Issue Overview

This PR addresses [#issue](https://github.com/bloom-housing/bloom/issues/2852)

- [x] This change addresses the issue in full

## Description
This pr handles 2 pieces:
1) handles the new front end for the flagged records (the table/general layout)
2) makes modifications to the flagged set, and applications table.
mainly it removes the status flag at the flagged set layer, and adds something similar to the applications table 

## How Can This Be Tested/Reviewed?
after a reseed there will be data present for flagged applications (duplicates)

go in and navigate to a listing's detail page, from there head the applications tab, and from there select pending reviews via the side nav (up to this point the work was done in other prs)

select one of these flagged sets from the table, you should be navigated to a new table of the dupicates. You should be able to check some number of them, and the data on this table should look like the mock attached to the ticketo

## Checklist:
- [x] My code follows the style guidelines of this project
- [x] I have added QA notes to the issue with applicable URLs
- [x] I have performed a self-review of my own code
- [x] I have reviewed the changes in a desktop view
- [x] I have reviewed the changes in a mobile view
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules
- [x] I have assigned reviewers
- [x] I have run `yarn generate:client` and/or created a migration if I made backend changes that require them
- [ ] I have exported any new pieces added to ui-components
- [x] My commit message(s) is/are polished, and any breaking changes are indicated in the message and are well-described
- [x] Commits made across packages purposefully have the same commit message/version change, else are separated into different commits

## Reviewer Notes:

Steps to review a PR:

- Read and understand the issue, and ensure the author has added QA notes
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Also review the acceptance criteria on the Netlify deploy preview (noting that these do not yet include any backend changes made in the PR)
- Either explicitly ask a clarifying question, request changes, or approve the PR if there are small remaining changes but the PR is otherwise good to go

## On Merge:

If you have one commit and message, squash. If you need each message to be applied, rebase and merge.

### CODE REVIEW

File: backend/core/src/application-flagged-sets/entities/application-flagged-set.entity.ts [hunk 1]
```diff
@@ -32,11 +32,6 @@ export class ApplicationFlaggedSet extends AbstractEntity {
   @Type(() => User)
   resolvingUser: User
 
-  @Column({ enum: FlaggedSetStatus, nullable: false, default: FlaggedSetStatus.flagged })
```
Comment (emilyjablonski) [inline backend/core/src/application-flagged-sets/entities/application-flagged-set.entity.ts:35]: Unrelated but above do you mind typing `rule` as the enum?

Comment (YazeedLoonat) [reply backend/core/src/application-flagged-sets/entities/application-flagged-set.entity.ts:35]: Hey @emilyjablonski I'm a lil confused on this comment, that line is removing the column from application flagged-set entity?

Comment (emilyjablonski) [reply backend/core/src/application-flagged-sets/entities/application-flagged-set.entity.ts:35]: It's a few lines above :) I can handle in a separate PR as well, just a typing issue I ran into

**Comment by emilyjablonski** [comment]
The description indicates `This change addresses the issue in full` just want to confirm that this is just a subset ya?

**Comment by emilyjablonski** [comment]
I'm good to handle these in the next PR that includes the modal/ability to change an application's status, just recording here for now:
* remove pointer cursor on a set name when the app is open and the resolving functionality is disabled
* save button should be disabled until something is selected
* last column string should be `application status` instead of `review status`

**Comment by emilyjablonski** [comment]
It also looks like this PR includes a rebase on dev and the new changes is that intended? It's making the diff a little hard to parse

**Comment by emilyjablonski** [comment]
@YazeedLoonat At the moment for me, clicking on an individual flagged set brings me to a blank white page (but I'm not seeing a clear error thrown)

**Comment by YazeedLoonat** [comment]
Hey @emilyjablonski got this updated if you wanna 👀 again please!

**Review by seanmalbert — CHANGES_REQUESTED** [review]
Hey @YazeedLoonat ,
Something doesn't seem right here. If I look at the Coliseum listing (after changing its status to closed), pending review count says 15, but there are only two sets available:

Screen Shot 2022-09-07 at 12 59 08 PM

Then, if I click into one of the sets, I see all 24 applications, and instead should see only the applications in that set.

I don't know if "resolved" is supposed to be part of this PR, but that has similar issues.

I think I could benefit from us pairing on a review of this, so let's get something scheduled.

File: backend/core/src/application-flagged-sets/application-flagged-sets-cronjob-consumer.ts [hunk 2]
```diff
@@ -97,7 +97,8 @@ export class ApplicationFlaggedSetsCronjobConsumer {
     const afsesToBeRemoved: Array<ApplicationFlaggedSet> = []
 
     for (const afs of afses) {
-      afs.status = FlaggedSetStatus.flagged
+      // TODO: update this so it resolves at the application level instead of the flagged set level
+      // afs.status = FlaggedSetStatus.flagged
```
[the remark this answers is not in the document]
Comment (YazeedLoonat) [reply backend/core/src/application-flagged-sets/application-flagged-sets-cronjob-consumer.ts:101]: nice catch here, I wasn't sure when I commented these out if another pr would need to come back through and clean them up

it seems like that isn't the case though so I'll remove em

File: backend/core/src/application-flagged-sets/application-flagged-sets.service.ts [hunk 3]
```diff
@@ -122,7 +144,8 @@ export class ApplicationFlaggedSetsService {
       // eslint-disable-next-line @typescript-eslint/no-unnecessary-type-assertion
       afs.resolvingUser = this.request.user as User
       afs.resolvedTime = new Date()
-      afs.status = FlaggedSetStatus.resolved
+      // TODO: update this so it resolves at the application level instead of the flagged set level
```
[the remark this answers is not in the document]
Comment (YazeedLoonat) [reply backend/core/src/application-flagged-sets/application-flagged-sets.service.ts:147]: this one is handled in https://github.com/bloom-housing/bloom/pull/3020 as the whole function needed to be updated

File: sites/partners/pages/listings/[id]/applications/pending/index.tsx [hunk 4]
```diff
@@ -199,6 +201,7 @@ const ApplicationsList = () => {
                 />
 
                 {/* <div className="flex flex-row justify-end">
+>>>>>>> upstream/feat/duplicates-v2
```
[the remark this answers is not in the document]
Comment (YazeedLoonat) [reply sites/partners/pages/listings/[id]/applications/pending/index.tsx:204]: nice catch ty!

**Comment by YazeedLoonat** [comment]
closing in favor of https://github.com/bloom-housing/bloom/pull/3020
